index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Ben Omayio</title>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@700&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@300&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="styles/main.css" />
+  </head>
+  <body>
+    <div class="container">
+      <main>
+        <nav>
+          <a href="#">Kiboma's</a
+          ><a><img src="imgs/showhide_buttons.png" alt="" /></a>
+        </nav>
+        <h1>I’m Omayio Glad to see you!</h1>
+        <p>
+          I’m a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hestiate to contact
+          me.
+        </p>
+        <h5>LET’S CONNECT</h5>
+        <ul class="flex">
+          <li><img src="imgs/Twitter.png" alt="Twitter Icon" /></li>
+          <li><img src="imgs/Linkedin-icon.png" alt="Linkedin-icon" /></li>
+          <li><img src="imgs/M.png" alt="M icon" /></li>
+          <li><img src="imgs/Github.png" alt="Github Icon" /></li>
+          <li><img src="imgs/hands.png" alt="hands" /></li>
+        </ul>
```

Comment:
### Requirement
- [ ] Nice to fix the design.  You should add a background image like in the design.
- [ ] To stick to the design you should change the text color, and font size. 

### ORIGINAL
![image](https://user-images.githubusercontent.com/38635158/164824038-75b5451e-0578-4e5c-80b8-75146ea60d4f.png)
____

### YOUR DESIGN
![image](https://user-images.githubusercontent.com/38635158/164822904-65b31b56-3360-40c6-90b0-3752ab3afaa0.png)



____

### OPTIONAL
_I write it as an optional but this is a requirement that is not mentioned in the first review. Please fix it._


- [ ] This part should be wrapped by a section tag. Please fix it.